agent/src/main/java/org/openremote/agent/protocol/tradfri/device/event/Event.java

```diff
@@ -0,0 +1,19 @@
+package org.openremote.agent.protocol.tradfri.device.event;
+
+import org.openremote.agent.protocol.tradfri.device.Device;
+
+/**
+ * The class that represents an event that occurred to an IKEA TRÅDFRI device
+ * @author Stijn Groenen
+ * @version 1.0.0
+ */
+public class Event {
```

Comment:
Empty class has no purpose and adds no value

Reply:
The Event class has been removed.
The Event class was used as a parent for the DeviceEvent class and the GatewayEvent class. Though it did not provide any functionality, it was used as a shared parent which was used to limit the generic type of the EventHandler class.